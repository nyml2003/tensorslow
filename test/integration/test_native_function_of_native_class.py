s = "this is a lowercase string"
print(s.upper())
