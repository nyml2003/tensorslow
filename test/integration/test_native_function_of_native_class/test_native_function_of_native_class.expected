THIS IS A LOWERCASE STRING
